src/app/layout/navbar/navbar.component.html

```diff
@@ -1,24 +1,26 @@
-<nav class="navbar-fixed">
-  <div class="nav-wrapper">
-    <div class="container">
-      <a href="#" class="brand-logo">GP4Tech</a>
-      <a href="#" data-target="mobile-menu" class="sidenav-trigger">
-        <i class="fa fa-bars"></i>
-      </a>
-      <ul class="left hide-on-med-and-down links">
-        <li>
-          <a>{{ 'links.about' | translate }}</a>
-        </li>
-      </ul>
-      <div class="right">
-        <app-language-button></app-language-button>
+<div id="app-menu">
+  <nav class="navbar-fixed">
+    <div class="nav-wrapper">
+      <div class="container">
+        <a href="#" class="brand-logo"><img src="assets/images/GP4Tech-logo.png"></a>
+        <a href="#" data-target="mobile-menu" class="sidenav-trigger">
+          <i class="fa fa-bars"></i>
+        </a>
+        <ul class="left hide-on-med-and-down links">
+          <li *ngFor="let view of views" [class.selected]="view === selectedView" (click)="onSelect(view)" >
+            <a routerLink="{{view.name}}">{{ view.i18n | translate | uppercase }}</a>
+          </li>
+        </ul>
+        <div class="right">
+          <app-language-button></app-language-button>
+        </div>
       </div>
     </div>
-  </div>
-</nav>
-
-<ul appSidenav class="sidenav" id="mobile-menu">
-  <li>
-    <a class="sidenav-close">{{ 'links.about' | translate }}</a>
-  </li>
-</ul>
+  </nav>
+  
+  <ul appSidenav class="sidenav" id="mobile-menu">
+    <li *ngFor="let view of views" [class.selected]="view === selectedView" (click)="onSelect(view)" >
+      <a routerLink="{{view.name}}">{{ view.i18n | translate | uppercase }}</a>
```

Comment:
If you are binding the routerLink, that isn't the correct notation. Please use: [routerLink]="view.name"

Reply:
done